backend/src/flotilla/api/robots_api.py

```diff
@@ -59,9 +51,7 @@ async def get_robots(
     db: Session = Depends(get_db),
     pagination_params: PaginationParams = Depends(),
 ) -> List[Robot]:
-    db_robots: List[RobotDBModel] = read_list_paginated(
-        db=db, params=pagination_params, modelType=RobotDBModel
-    )
+    db_robots: List[RobotDBModel] = read_robots(db=db)
```

Comment:
Kind of forgot to add it again after removing for some testing. Don't think it is very necessary for robots though, since there will be a limited number of robots on an asset